src/meshdb/settings.py

```diff
@@ -41,6 +41,7 @@
     "db.mesh",
     "db.nycmesh.net",
     "127.0.0.1",
+    "10.0.2.2",
```

Comment:
Why do we want to allow this?

Reply:
oops that's an artifact of my testing on windows.